<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DodeIDE</class>
 <widget class="QMainWindow" name="DodeIDE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DodeIDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="CodeTextArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>891</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="OutputArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>891</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CompileButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>COMPILE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
